service_crategen/src/commands/generate/codegen/mod.rs

```diff
@@ -208,6 +208,12 @@ where
             }}
         }}
 
+        impl fmt::Debug for {type_name} {{
+            fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {{
+                write!(f, \"{type_name} {{{{ region: {{:?}} }}}}\", &self.region)
```

Comment:
It'd be better to use [`Formatter::debug_struct`](https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.debug_struct) here, so that the human-readable `{:#?}` works as expected.

Reply:
Thanks for the feedback @iliana , I wasn't aware of this method :tada: